schema.sql

```diff
@@ -11,4 +11,9 @@ CREATE TABLE species (id INT GENERATED ALWAYS AS IDENTITY, name VARCHAR(45) NOT
  ALTER TABLE animals ADD COLUMN id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY;
  ALTER TABLE animals DROP COLUMN species;
   ALTER TABLE animals ADD COLUMN species_id INT, ADD CONSTRAINT fk_species_id FOREIGN KEY(species_id) REFERENCES species(id);
-   ALTER TABLE animals ADD COLUMN owner_id INT, ADD CONSTRAINT fk_owner_id FOREIGN KEY(owner_id) REFERENCES owners(id);
\ No newline at end of file
+   ALTER TABLE animals ADD COLUMN owner_id INT, ADD CONSTRAINT fk_owner_id FOREIGN KEY(owner_id) REFERENCES owners(id);
+
+--    JOIN and RELATIONSHIPS
+CREATE TABLE vets (ID int GENERATED ALWAYS AS INDENITY, name VARCHAR NOT NULL, age INT NOT NULL, date_of_graduation date NOT NULL, PRIMARY KEY(id) );
```

Comment:
- [ ] [OPTIONAL] I would strongly recommend you add size to your [PostgreSQL Character Types](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-char-varchar-text/#:~:text=The%20advantage%20of%20specifying%20the,the%20VARCHAR(n)%20column.), The advantage of specifying the length specifier for the VARCHAR data type is that PostgreSQL will issue an error if you attempt to insert a string that has more than n characters into the VARCHAR(n) column. Please fix it 👍